### ISSUE TITLE
Ability to specify analyticsUserName

### ISSUE THREAD

**Issue by anthonycastelli** [vapor-community/swiftybeaver-provider#12]
Would it be possible to specify the `analyticsUserName` per log? I'd like to be able to do something like this 
```swift
let logger = try req.make(SwiftyBeaverLogger.self)
logger.analyticsUserName = "User: \(user.email)"
logger.info("Authenticated \(user.name) \(user.email)")
```

**Comment by gperdomor** [comment]
Yeah, is a property of SBPlatform but is not handled in the provider initialization... I will find some time tomorrow to update this :D

**Comment by gperdomor** [comment]
@anthonycastelli check this PR https://github.com/vapor-community/swiftybeaver-provider/pull/13

**Comment by gperdomor** [comment]
Fixed in #13
